lib/semian.rb

```diff
@@ -124,7 +124,8 @@ def register(name, tickets:, permissions: 0660, timeout: 0, error_threshold:, er
       error_threshold: error_threshold,
       error_timeout: error_timeout,
       exceptions: Array(exceptions) + [::Semian::BaseError],
-      implementation: ::Semian::Simple,
+      permissions: permissions,
+      implementation: ::Semian::SysV,
```

Comment:
Yeah, I don't quite understand how the implementation can be hardcoded. How does the fallback work then? It sounds like in that case there's no test to make sure it goes to the Simple fallback when SysV is not available / disabled.